### PR TITLE
Add vaccination information

### PR DESCRIPTION
**Pull request by divyagar**
Co-authored-by: Amit <amitkr0819@mgmail.com>
Co-authored-by: Khavin <khavinshankar@gmail.com>

https://github.com/coronasafe/care_fe/issues/945

### CODE REVIEW

**Review by afzalIbnSH — APPROVED** [review]
LGTM

File: care/facility/api/serializers/patient.py [hunk 1]
```diff
@@ -201,6 +201,13 @@ def validate(self, attrs):
             raise serializers.ValidationError(
                 {"non_field_errors": [f"Either age or date_of_birth should be passed"]}
             )
+
+        if(validated["is_vaccinated"]):
+            if(validated["number_of_doses"] == 0):
+                raise serializers.ValidationError("Number of doses cannot be 0")
+            if(validated["vaccine_name"] == None):
```
Comment (vigneshhari) [inline care/facility/api/serializers/patient.py:208]: Is the choice validation already done ?

**Review by vigneshhari — COMMENTED** [review]
Changes Required

**Comment by vigneshhari** [comment]
Resolve Conflicts as well, DO NOT FORCE PUSH.

File: care/facility/api/serializers/patient.py (hunk 1 above)
Comment (vigneshhari) [reply care/facility/api/serializers/patient.py:213]: @divyagar please resolve the question if its completed

**Comment by vigneshhari** [comment]
is_vaccinated is already a field in the patient

**Comment by divyagar** [comment]
Yes I have not added is_vaccinated field.

**Comment by vigneshhari** [comment]
My bad, i was looking at additions only.

**Pull request merged**